Add Xuening to roster_names.md

Xuening99 opened the pull request:
#5

mitchphillipson commented:
This could use a better title. Like "Add Xuening to roster_names.md"

mitchphillipson commented on roster_names.md:
```diff
@@ -1,3 +1,7 @@
\ No newline at end of file
+| Mitch Phillipson | 
+
+| Name |
+| ---  |
```
You don't need these two lines. They will make a new table. Remove these and put your name under mine

mitchphillipson commented:
Why do you have two PRs? Try to resolve the conflicts on this one.

Xuening99 commented:
I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you

mitchphillipson commented:
![image](https://github.com/mitchphillipson/aae718/assets/19531741/8831098f-7244-4ad3-9dd6-0c3bbba1f093)

This is what I see in the merge conflict. Click the `Resolve conflicts` button and delete the `<<< main` and all lines after you name. Then click the `mark as resolved` button.

mitchphillipson commented:
> I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you

I'm making this a little more challenging than it is normally. You're doing just fine.

Xuening99 commented:
> > I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you
> 
> I'm making this a little more challenging than it is normally. You're doing just fine.



> > I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you
> 
> I'm making this a little more challenging than it is normally. You're doing just fine.



> > I have changed the code in this one and have the updated ones in commit and the file changed should show my changes. I also closed the second pull request. Hope I can get this right this time. Thank you
> 
> I'm making this a little more challenging than it is normally. You're doing just fine.

Thank you! And I also resolved the conflict now on my end.

mitchphillipson commented:
I still see the conflict, did you commit after resolving the conflict?

Xuening99 commented:
> I still see the conflict, did you commit after resolving the conflict?

Sorry, I didn't commit the first time. Does it work now?

mitchphillipson commented:
Not only did it work, I'm going to merge and resolve your issue. You're done!

Xuening99 commented:
> Not only did it work, I'm going to merge and resolve your issue. You're done!

Thank you!
